src/drivers/mct/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<namelist_defaults>
+
+<drv_flds_in_files>Buildconf/camconf/drv_flds_in,Buildconf/clmconf/drv_flds_in</drv_flds_in_files>
+
+</namelist_defaults>
```

Comment:
There  is **not** be a drivers/mct/bld directory at this point - this needs to be removed. All of the files there have older schemas that are no longer used and were removed when the driver namelist was pythonized. The old bld/ directory was removed as part of the pythonization and everything needed to configure and build the driver is in its cime_config/ directory.

Also - the bld/namelist_defaults_drv.xml was refactored to the new python entry_id schema and is now the file drivers/mct/cime_config/namelist_definition_drv_flds.xml